index.html

```diff
@@ -0,0 +1,227 @@
+    <div class="menu-mobile">
+      <a class="buttom" href=#modal>
+        <img src="./img/icon-mobile.png">
+      </a>
+      <div id="modal" class="overlay">
+        <a class="cancel" href="#"></a>
+        <div class="popup">
+          <h1 class="popup-title">WoodMood</h1>
+          <a class="close" href="#">&times;</a>
+          <div class="popup-container">
+            <p class="popup-content"> About</p>
+            <p class="popup-content"> Services</p>
+            <p class="popup-content"> Gallery</p>
+            <p class="popup-content"> Book a Consultation</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <nav class="desktop-menu">
+      <p>About us</p>
+      <p>Services</p>
+      <p>Gallery</p>
+      <p><a href="#faq">FAQ</a></p>
+
+    </nav>
+    <h1 class="main-title">WoodMood</h1>
+
+    <div class="content">
+      <div class="text-content">
+        <h1 class="subtitle"> Designed for life</h1>
+        <p class="short-text"> Furniture and objects crafted from source hardwoods. </p>
+      </div>
+      <div class="button-content">
+        <a class="form-buttom" href="#form">Book a Consultation</a>
+      </div>
+    </div>
+  </section>
+  <!-- Company information. -->
+  <secion class="woodmood-about">
+    <div class="about-us">
+      <div class="text-content2">
+        <h1 class="subtitle2">Generations of experience crafting wood</h1>
+        <p class="short-text2">Founded in 1921, WoodMood is a fourth-generation family business dedicated to the art and
+          craft of fine carpentry.e approach every project as a unique opportunity to apple classic woodworking
+          techniques to modern aesthetics.</p>
+      </div>
+    </div>
+    <div class="wooodmood-img">
+      <img src="./img/img-sect-2.jpg">
+    </div>
+    </div>
+  </secion>
+
+  <!-- Book a Consultation form -->
+  <!-- Signup form (Personal data  -->
+  <section class="form" id="form">
+    <form action="/action_page.php">
+
+      <fieldset>
+        <legend>User data</legend>
+        <label class="label" for="fname">Firts Name</label><br>
+        <input class="box" type="text" id="fname" name="fname" required><br>
+
+        <label class="label" for="lname"> Last Name</label><br>
+        <input class="box" type="text" id="lname" name="lname" required><br>
+
+        <label class="label" for="email">Email</label><br>
+        <input class="box" type="email" id="email" name="email" required><br>
+
+        <label class="label" for="pass">Phone Number</label><br>
+        <input class="box" type="tel" id="phone" name="phone" placeholder="123-456-7890"
+          pattern="[0-9]{3}[0-9]{3}[0-9]{4}" required minlength="8" maxlength="12"><br>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Subject of your Consultation</legend>
+        <div class="themes-options">
+          <div class="option">
+            <input class="radio" type="radio" id="furniture" value="furniture">
```

Comment:
When it comes to radio buttons, there is one thing that we should be super aware of : user should be allowed to select only one radio button at a time. In your case however, I was able to select all of them.
How can we fix that? We need to add `name` attribute to all of them, moreover, this attribute has to have same value for all radio buttons, like this :

```
<input type="radio" name="theme" value="furniture" />
<input type="radio" name="theme" value="floaring" />
<input type="radio" name="theme" value="illustration" />
<input type="radio" name="theme" value="mobile-art" />
```

Reply:
Hey, @EmOlsz I've done the changes but after pushing them, I don't see my changes in the PR. Also, it says that the PR is closed. Maybe is because of that?

Reply:
Hey, indeed, it's closed, you closed it 3 days ago :D If you will reopen it, then we should see the changes!
<img width="1049" alt="Screenshot 2021-02-10 at 11 53 41" src="https://user-images.githubusercontent.com/22129355/107506931-1d458700-6b9f-11eb-9bcb-eb71efd47e59.png">

Reply:
HI @EmOlsz 

I didn't realize that was me who closed the PR. Now it's ready.